shared/osarch_test.go

```diff
@@ -51,6 +31,11 @@ func TestGetArch(t *testing.T) {
 			"x86_64",
 			"amd64",
 		},
+		{
+			"debian",
+			"s390x",
+			"s390x",
```

Comment:
Why the duplicate entry? :)

Reply:
The first value is the provided value, the second one is the expected value.

Reply:
Ah, ok.